src/index.html

```diff
@@ -9,7 +9,49 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST" >
+    <fieldset class="form-fields">
+      <legend>Personal information</legend>
+      <label class="input-field">Surname: <input type="text" name="surname" autocomplete="off"></label>
+      <label class="input-field">Name: <input type="text" name="name" autocomplete="off"></label>
+      <label class="input-field">How old are You? <input type="number" name="age" min="1" max="99" value="12" class="age"></label>
+      <label class="input-field">Full date of birth: <input type="date" name="birth"></label>
+      <label class="input-field">I accept the term of the agreement <input type="checkbox" name="agreement"></label>
+    </fieldset>
+    <fieldset class="form-fields">
+      <legend>Registration</legend>
+      <label class="input-field">E-mail: <input type="email" name="email" placeholder="email@example.com"></label>
+      <label class="input-field">Password: <input type="password" name="password" minlength="6"></label>
+    </fieldset>
+    <fieldset class="form-fields">
+        <legend>An interesting fact about you!</legend>
+        <div class="input-field">Do you love cats?
+          <label ><input type="radio" name="cats">Yes</label>
+          <label ><input type="radio" name="cats">No</label>
```

Comment:
```suggestion
          <label><input type="radio" name="cats">No</label>
```
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.